cmake/SetupDoxygen.cmake

```diff
@@ -1,10 +1,29 @@
 # Distributed under the MIT License.
 # See LICENSE.txt for details.
 
+# Targets to preprocess the documentation
+# - Convert all `.ipynb` files in `docs/` to markdown, so they get picked up by
+#   Doxygen.
+get_filename_component(_PY_BIN_DIR ${Python_EXECUTABLE} DIRECTORY)
```

Comment:
When I try to run this on my machine, cmake fails with
```
CMake Error at cmake/SetupDoxygen.cmake:7 (get_filename_component):
  get_filename_component called with incorrect number of arguments
Call Stack (most recent call first):
  CMakeLists.txt:53 (include)
```

Reply:
Maybe `Python_EXECUTABLE` is empty for some reason? I'll fix it.

Reply:
should be fixed now.

Reply:
Yep.  Now it works.